productImport/productImportHandler.go

```diff
@@ -123,7 +123,7 @@ func (ph *ProductImportHandler) runProductValidationImportFlow(columnMap map[str
 			}
 		}
 	} else if len(sources) == 0 {
-		return fmt.Errorf("SOURCE IS NOT FOUND")
+		return fmt.Errorf("PRODUCTS SOURCE WAS NOT FOUND")
```

Comment:
I think this message is misleading - **products source** exists and was successfully read but it is empty.